Tests/Components/ApplePayDirect/Services/ApplePayPaymentMethodTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace MollieShopware\Tests\Components\ApplePayDirect\Services;
+
+use MollieShopware\Components\ApplePayDirect\Services\ApplePayPaymentMethod;
+use MollieShopware\Components\Services\PaymentMethodService;
+use PHPUnit\Framework\TestCase;
+
+class ApplePayPaymentMethodTest extends TestCase
+{
+    private $paymentMethodService;
+
+    public function setUp(): void
+    {
+        $this->paymentMethodService = new ApplePayPaymentMethod(
+            $this->createMock(PaymentMethodService::class)
+        );
+
+        require_once __DIR__ . '/../../../Shopware/Components/Plugin.php';
+    }
+
+    /**
+     * Tests if isApplePayPaymentMethod will return false if given method name is not name of Apple Pay method
+     *
+     * @covers \MollieShopware\Components\ApplePayDirect\Services\ApplePayPaymentMethod
+     */
+    public function testIsApplePaymentMethodWillReturnFalseOnOtherMethod()
+    {
+        $this->assertFalse($this->paymentMethodService->isApplePayPaymentMethod('mollie_creditcard'));
```

Comment:
could you use data providers please and do only 1 assertion - thank you

Reply:
Done